docs/content/_navigation.json

```diff
@@ -1055,6 +1055,10 @@
             "title": "Structuring your project",
             "path": "/guides/dagster/recommended-project-structure"
           },
+          {
+            "title": "Using tags to attach metadata",
```

Comment:
"metadata" and "tags" are two separate things in Dagster, so I think we'd probably just want to call this something like "Using tags"

Reply:
Works for me!